inventory/hosts.example

```diff
@@ -752,6 +752,19 @@ debug_level=2
 #
 # Option C - none -- Prometheus, alertmanager and alertbuffer will use emptydir volumes
 # which are destroyed when pods are deleted
+#
+# Deploying Prometheus in a offline / disconnected installation, requires the following addition. 
```

Comment:
Why are these required? Those are the defaults for `deployment_type=openshift-enterprise`

Reply:
If not set the setting will be registry.access.redhat.com but if your offline / disconnected it has to be pointed towards your internal registry, where disconnected / offline seriously lack in documentation. 

As far as i can find it does not respond to oreg_url.

Reply:
I'm sorry. I still don't understand.
The params you added to the inventory file are identical to the default ones:
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/vars/openshift-enterprise.yml

So why is that addition required?

Reply:
If you want to install prometheus in a disconnected environment, you need to change those variables to yoru private registry, and as they are not documented anywhere in case you need to change from the default registry.access.redhat.com to lets say docker-registry-default.something.example.com you need to dig trough the playbooks, after you have already run trough them the first time where everything will fail cause you cant reach registry.access.redhat.com

Reply:
Ok. So the new example wasn't clear to me.
Should be something like:
```
# Deploying Prometheus in an offline / disconnected installation requires setting offline image prefix. 
# Each image has its own prefix and version, version can be left out if going with the default Openshift version.
# For example if the offline repo is: docker-registry-default.something.example.com, the following setting should be applied:
# openshift_prometheus_image_prefix=docker-registry-default.something.example.com/openshift3/
```
and all the other images example...